### PR TITLE
Add new amp-analytics vendor: New Relic

### LINKED ISSUE #11463: Intent To Implement: Adding New Relic config to amp-analytics
I'll follow up with a PR that follows the steps outlined in the [documentation](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/integrating-analytics.md#adding-your-configuration-to-the-amp-html-runtime).

### PR DESCRIPTION
**Pull request by blainesch**
This adds New Relic to the amp-analytics provider list.

Fixes #11463

### CODE REVIEW

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -1007,6 +1007,49 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
...
+        '&timeToRedirectStart=${navTiming(redirectStart)}' +
+        '&timeToRedirectEnd=${navTiming(redirectEnd)}' +
+        '&timeToSecureConnection=${navTiming(secureConnectionStart)}' +
+        '&timestamp=${timestamp}' +
+        '&ampVersion=${ampVersion}' +
+        '&pageLoadTime=${pageLoadTime}',
+    },
+    'vars': {
+      'beacon': 'bam.nr-data.net',
+      'appId': [''],
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors.js]: nit: `[]` is good enough.

Comment (blainesch) [reply extensions/amp-analytics/0.1/vendors.js]: Updated. Thanks!

**Comment by blainesch** [comment]
@zhouyx Thanks for the quick review! I'm just waiting on the legal time to give me the thumbs up on signing that doc.

**Comment by zhouyx** [comment]
@blainesch Did your company sign the CLA? If so please reply your company name to this thread after the doc is signed. Thanks.

**Comment by blainesch** [comment]
I signed it!

**Comment by zhouyx** [comment]
@blainesch You sign the CLA as an individual contributor right? If that's the case googlebot should be able to pick that up fast.
Did you check that the your email address match?
> If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address. Check your existing CLA data and verify that your email is set on your git commits.

If so my best suggestion would be to rebase, it sometimes fix the CLA. 
Thanks.

**Comment by blainesch** [comment]
@zhouyx I signed it as an employer. I created this commit using my work email address, which is part of the google group I signed the doc with.

**Comment by blainesch** [comment]
I signed it!

**Comment by zhouyx** [comment]
@blainesch If your company signed the CLA, that may take longer. Can you provide the name of your company and we can see if it has been processed or not.

**Comment by blainesch** [comment]
@zhangsu it's `New Relic, Inc`

**Comment by zhouyx** [comment]
@blainesch I am not able to locate New Relic, Inc on the list. Please note that it typically takes a few days to process the corporate CLA. It you think it's taking longer than expected, please reach out to check status. Thanks!

**Comment by blainesch** [comment]
I'll check back later next week. Thank you!

**Comment by blainesch** [comment]
@zhouyx any update on the cla?

**Comment by blainesch** [comment]
@zhouyx any update on the cla?

**Comment by zhouyx** [comment]
Hmmm I am still not able to find the company name. Just sent out an email to check the status, I'll let you know when I hear back from them. Thanks

**Comment by zhouyx** [comment]
@blainesch Some update: It appears that New Relic corporate CLA is still under review because it has not yet been signed by the New Relic's representative. 
If you happen to know the contact person, please reach out to him/her. Thank you!

**Comment by blainesch** [comment]
I saw the form being filled out, are you sure? I can ask them to resubmit it...

**Comment by blainesch** [comment]
new_cla_-_google_cla

**Comment by blainesch** [comment]
Oh, it emailed them and they thought it was spam. They found it and signed it from the email.

**Comment by zhouyx** [comment]
@blainesch Good News! New Relic Inc now appears in the accepted corporate CLAs list. Please reply to this thread to enable googlebot to verify : )

**Comment by blainesch** [comment]
I signed it!

**Comment by zhouyx** [comment]
@blainesch Thanks for the Pull Request! Merged

**Comment by blainesch** [comment]
@zhouyx thanks for helping me! I have another question if you have time. I tried to use the snippet and I'm getting an error. You can see an example in this repo:

https://github.com/blainesch/amp/blob/short-amp-snippet/analytics-short.html#L27-L37

Here's a screenshot of the error I'm getting:

analytics-short_html_and_1___usr_local_bin_tmux__tmux_

Any advice on why this isn't working would be appreciated. Thank you!

**Comment by zhouyx** [comment]
Hey @blainesch. You are testing against our PROD version which doesn't have your vendor config included yet. That's why amp-analytics can't find a request string.

This PR was merged on Thursday, didn't make to this week's canary cut. Unfortunately you can only test it locally right now. After next week's canary cut, your config will be available  by opt into our [dev channel ](https://cdn.ampproject.org/experiments.html). 

The change is expected to roll out to prod one week after canary (target at around 11/15)

**Comment by blainesch** [comment]
Thank you [:

**Pull request merged**